CHANGES.rst

```diff
@@ -258,6 +258,9 @@ Bug Fixes
 
   - Handle sorting NaNs and masked values in jsviewer. [#4052, #5572]
 
+  - Ensure mixin columns can be set to scalars if the type supports
```

Comment:
This should be more like 
```
- Ensure mixin columns can be added to a table using a scalar value for the
  right-hand side if the type supports broadcasting.  For example, for an existing
  ``QTable`` now ``t['q'] = 3*u.m`` will add a column ``q`` as expected.
```
I.e. this is about *adding* a column, not *setting* an existing column (which already works if the mixin type supports it).